README.md

```diff
@@ -23,32 +25,31 @@ Code. Ship. Scale.
 
 <p align="center">
   <a href="https://github.com/flyteorg/flyte/releases/latest">
-    <img src="https://img.shields.io/github/release/flyteorg/flyte.svg" alt="Current Release" />
-  </a>
+    <img src="https://img.shields.io/github/release/flyteorg/flyte.svg" alt="Current Release" /></a>
   <a href="https://github.com/flyteorg/flyte/actions/workflows/sandbox.yml">
-    <img src="https://github.com/flyteorg/flyte/actions/workflows/sandbox.yml/badge.svg" alt="Sandbox Build" />
-  </a>
+    <img src="https://github.com/flyteorg/flyte/actions/workflows/sandbox.yml/badge.svg" alt="Sandbox Build" /></a>
   <a href="https://github.com/flyteorg/flyte/actions/workflows/tests.yml">
-    <img src="https://github.com/flyteorg/flyte/actions/workflows/tests.yml/badge.svg" alt="End-to-End Tests" />
-  </a>
+    <img src="https://github.com/flyteorg/flyte/actions/workflows/tests.yml/badge.svg" alt="End-to-End Tests" /></a>
   <a href="http://www.apache.org/licenses/LICENSE-2.0.html">
-    <img src="https://img.shields.io/badge/LICENSE-Apache2.0-ff69b4.svg" alt="License" />
-  </a>
-  <img src="https://img.shields.io/github/commit-activity/w/flyteorg/flyte.svg?style=plastic" alt="Commit Activity" />
-  <img src="https://img.shields.io/github/commits-since/flyteorg/flyte/latest.svg?style=plastic" alt="Commits since Last Release" />
-  <img src="https://img.shields.io/github/milestones/closed/flyteorg/flyte?style=plastic" alt="GitHub Milestones Completed" />
-  <img src="https://img.shields.io/github/milestones/progress-percent/flyteorg/flyte/18?style=plastic" alt="GitHub Next Milestone Percentage" />
+    <img src="https://img.shields.io/badge/LICENSE-Apache2.0-ff69b4.svg" alt="License" /></a>
+  <a href="https://github.com/flyteorg/flyte/commits">
+    <img src="https://img.shields.io/github/commit-activity/w/flyteorg/flyte.svg" alt="Commit Activity" /></a>
```

Comment:
This formatting somehow doesn't look clean. Is the anchor tag on the new line the reason for underscores?

Reply:
Yes, everything before the closing anchor tag is interpreted as text, including newlines. By default the style is underlined and we cannot change that on github. In alternative we could put everything on the same line
`<a href="https://github.com/flyteorg/flyte/commits"><img src="https://img.shields.io/github/commit-activity/w/flyteorg/flyte.svg" alt="Commit Activity" /></a>`
but I thought it would make more sense to have the `img` tag on a new line even if the closing anchor doesn't have the same indentation as the opening one

Reply:
Can you try replacing anchor tags with https://stackoverflow.com/a/61072867/9695360 and check if this works?

Reply:
We wouldn't  be able to keep them centered if we go for markdown. The closest solution to mixing both is something similar to this https://stackoverflow.com/questions/29368902/how-can-i-wrap-my-markdown-in-an-html-div but the only difference would be using markdown image tags instead of html `img` tags. They all still have to stay on one line